src/js/admin.js

```diff
@@ -432,6 +432,45 @@ WebcomAdmin.fileUploader = function($) {
     .on('click', '.meta-file-delete', removeFile);
 };
 
+WebcomAdmin.collegesUpload = function($) {
```

Comment:
At this point, since we're expecting this to be a fairly temporary solution, I think leaving the separate function is okay. Making the file upload function work for this would require updating it a little to be more general (right now it relies on the #post-body id to be present, which isn't present on tax terms). I would imagine in the redesign of the main site theme, we'd handle this with ACF, eliminating this issue.